extras/tests/unitTest/unitTest.ino

```diff
@@ -373,23 +378,23 @@ test(timer_size) {
 
     Timer<0x2, CLOCK::millis, Task *> timer;
 
-    assertEqual(timer.size(), 0UL);
+    assertEqual((unsigned long) timer.size(), 0UL);
```

Comment:
I run my test on an Arduino Nano.  I got an error because `size_t` is NOT `unsigned long`.  This cast should make the test work on all platforms.